src/main/telemetry/smartport.c

```diff
@@ -739,7 +739,7 @@ void handleSmartPortTelemetry(void)
             //case FSSP_DATAID_A3         :
             case FSSP_DATAID_A4         :
                 if (feature(FEATURE_VBAT))
-                    smartPortSendPackage(id, vbat * 10 / batteryCellCount ); // given in 0.1V, convert to volts
```

Comment:
I'm pretty sure the comment is wrong. If vbat resolution is 100mV then `vbat * 10 / batteryCellCount` gives the individual cell voltage in 10mV increments not in 1V increments.